test/integration/buildah/buildah_test.go

```diff
@@ -0,0 +1,271 @@
+// +build integration
+
+package buildah
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/openshift/source-to-image/pkg/api/constants"
+	buildahpkg "github.com/openshift/source-to-image/pkg/buildah"
+	"github.com/openshift/source-to-image/pkg/docker"
+	"github.com/openshift/source-to-image/pkg/util/fs"
+)
+
+const (
+	busyboxImage = "docker.io/busybox:latest"
+	exampleImage = "nginx-centos7:testing"
+)
+
+func TestBuildah(t *testing.T) {
```

Comment:
cool ... that said, I do not think we should refrain from unit tests for pkg/buildah/buildah.go just cause these are here

Reply:
The tests are moved to this location, as commented before. This was done because of CI settings, where we only have `buildah` and `docker` for integration tests, and only in a few selected locations.